euth/comments/templatetags/react_comments.py

```diff
@@ -13,6 +15,10 @@
 def react_comments(context, obj):
     request = context['request']
 
+    serializer = ThreadSerializer(
+        obj.comments.all(), many=True, context={'request': request})
```

Comment:
I think that wouldn't be to wired to request. If it turns out to be a problem at some point, than we can add an if here.